File: SoLLDienTu_Server/SoLLDienTu_Server/SoLLDienTu/Models/MonHoc.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace SoLLDienTu.Models
{
    public partial class MonHoc
    {
        public MonHoc()
        {
            Gvmhs = new HashSet<Gvmh>();
            KetQuas = new HashSet<KetQua>();
        }

        public string MaMh { get; set; }
        public string TenMh { get; set; }
        public int? SoTc { get; set; }
        public int? Kyhoc { get; set; }

        public virtual ICollection<Gvmh> Gvmhs { get; set; }
        public virtual ICollection<KetQua> KetQuas { get; set; }
    }
}

File: SoLLDienTu/SoLLDienTu/app/src/main/java/com/example/solldientu/object/MonHoc.java
package com.example.solldientu.object;

public class MonHoc {
    private String maMh, tenMh;
    private int soTc, kyhoc;

    public MonHoc(String maMh, String tenMh, int soTc, int kyhoc) {
        this.maMh = maMh;
        this.tenMh = tenMh;
        this.soTc = soTc;
        this.kyhoc = kyhoc;
    }

    public MonHoc(String tenMh, int soTc, int kyhoc) {
        this.tenMh = tenMh;
        this.soTc = soTc;
        this.kyhoc = kyhoc;
    }

    public MonHoc() {
    }

    public String getMaMh() {
        return maMh;
    }

    public void setMaMh(String maMh) {
        this.maMh = maMh;
    }

    public String getTenMh() {
        return tenMh;
    }

    public void setTenMh(String tenMh) {
        this.tenMh = tenMh;
    }

    public int getSoTc() {
        return soTc;
    }

    public void setSoTc(int soTc) {
        this.soTc = soTc;
    }

    public int getKyhoc() {
        return kyhoc;
    }

    public void setKyhoc(int kyhoc) {
        this.kyhoc = kyhoc;
    }
}
